14_functions_lab/01.py

```diff
@@ -4,4 +4,30 @@
     mymul - returns the multiplication of its input arguments
     Ignore non-numeric arguments
 """
+import operator 
 
+def mysum(*paramsin):
+    res = 0
```

Comment:
A better way to check for numeric value is isinstance(n, numbers.Number). That also checks for floats